animation.go

```diff
@@ -0,0 +1,61 @@
+package fyne
+
+import "time"
+
+// AnimationCurve represents an animation algorithm for calculating the progress through a timeline.
+// Custom animations can be provided by implementing the "func(float32) float32" definition.
+// The input parameter will start at 0.0 when an animation starts and travel up to 1.0 at which point it will end.
+// A linear animation would return the same output value as is passed in.
+type AnimationCurve func(float32) float32
+
+var (
+	// AnimationEaseInOut is the default easing, it starts slowly, accelerates to the middle and slows to the end.
```

Comment:
AFAICS, the default is `nil` which is translated to `AnimationLinear` by the GLFW driver. Therefore, this seems to be wrong.

Reply:
Great catch thanks